### PR TITLE
Add Podcasts section to Community

### PR DESCRIPTION
**Pull request by olivierlacan**
This is definitely a little self-serving since I just launched
http://rubyfacets.com/ today but I've listed the other existing Ruby podcasts
that I know are still publishing new episodes at the moment. For now that
includes Ruby Rogues and the Ruby on Rails Podcast but I'm sure others will
find more relevant podcasts in languages other than English.

### CODE REVIEW

**Comment by stomar** [comment]
I'm generally in favour of listing some podcasts, but I still have some objections with this PR:

* I would rather wait some time before including a freshly launched podcast; but I'm ok with the other 2
* please amend the commit message to include only the necessary information (including the `en` tag); duplicating the complete PR description, including information on personal motivation etc., is not necessary
* please use named link references instead of numbers, see the Wiki

File: en/community/index.md [hunk 1]
```diff
@@ -42,6 +42,11 @@ to start:
   work-in-progress, discuss the future of Ruby, and welcome newcomers to
   the Ruby community.
 
+[Podcasts](podcasts/)
+: There's a lot of text written about Ruby on the web but there's also a lot of
+  audio: some podcasts cover new Ruby or RubyGem releases, others are
```
Comment (stomar) [inline en/community/index.md]: RubyGem releases -> gem releases

**Comment by colby-swandale** [comment]
@olivierlacan are you able to address the feedback? If not i can look into getting this merged.

**Comment by olivierlacan** [comment]
@stomar @colby-swandale Only took 2 years but addressed all issues and removed reference to the (now defunct) Ruby Facets podcast.

**Comment by stomar** [comment]
Thank you and sorry for the delay! Merged.

**Pull request merged**